### PR TITLE
Add merge request participants api

### PR DESCRIPTION
**Pull request by almorgv**
Add new `GetMergeRequestParticipants` method in `MergeRequestsService` to retreive merge request participants from API as described in https://docs.gitlab.com/ee/api/merge_requests.html#get-single-mr-participants

### CODE REVIEW

File: merge_requests.go [hunk 1]
```diff
@@ -399,6 +399,31 @@ func (s *MergeRequestsService) GetMergeRequestChanges(pid interface{}, mergeRequ
 	return m, resp, err
 }
 
+// GetMergeRequestParticipants gets a list of merge request participants
```
Comment (svanharmelen) [inline merge_requests.go:402]:
```suggestion
// GetMergeRequestParticipants gets a list of merge request participants.
```

File: merge_requests.go [hunk 2]
```diff
@@ -399,6 +399,31 @@ func (s *MergeRequestsService) GetMergeRequestChanges(pid interface{}, mergeRequ
...
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/merge_requests/%d/participants", pathEscape(project), mergeRequest)
+
+	req, err := s.client.NewRequest("GET", u, nil, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var p []*BasicUser
```
Comment (svanharmelen) [inline merge_requests.go:418]: Maybe add an `s` to indicate it are multiple participants?

Comment (almorgv) [reply merge_requests.go:418]: You mean change the variable name to `ps`?
```suggestion
	var ps []*BasicUser
```

Comment (svanharmelen) [reply merge_requests.go:418]: Yes indeed... Not a biggie, but would be nice.

Comment (almorgv) [reply merge_requests.go]: Done [de6dd49](https://github.com/xanzy/go-gitlab/pull/858/commits/de6dd49087b5822a4f01bbd74dddd0bb32a39760)

**Pull request merged**